docs/user/orion.rst

```diff
@@ -0,0 +1,92 @@
+- The Artemis exercise paths. Orion will suggest to store newly downloaded exercises at *default path/course/exercise name*, with the default path dependant of the setting.
+- The default commit message
+- An option to change the user agent if needed as well as a button to restart the integrated browser. If Orion gets stuck restarting the browser might solve the issue without restarting IntelliJ.
+
+Problems
+^^^^^^^^
+
+Please report any issues on `GitHub <https://github.com/ls1intum/Orion>`_.
+
+Orion for Students
+------------------
+
+Students can participate in programming exercises via Orion by performing the following steps:
+
+ 1. Navigate to the exercise using the integrated browser.
+ 2. After starting the exercise, a button to |open-in-intellij-button| appears.
+ 3. After cloning, the exercise can be solved in IntelliJ.
+ 4. Orion provides a button to |submit-button|. This will automatically commit and push the local changes and display the result of the automatic tests in IntelliJ.
+
+`Gif showcasing the usage for students <https://github.com/ls1intum/Orion#example-usage>`_
+
+Orion for Instructors
+---------------------
+
+Instructors can set up programming exercises via Orion by performing the following steps:
+
+ 1. The exercise needs to be created as described at the programming exercise creation step 1 and 2.
+ 2. After the creation, navigate to the instructor exercise overview using the integrated browser
+ 3. Each programming exercise will provide a button to |edit-in-intellij-button|. The button is rightmost in the table an might require scrolling. This will download the template, solution and test repository of the exercise
+ 4. Edit the repository files in IntelliJ.
+ 5. Orion provides a button to |submit-button|. This will commit and push all local changes to their respective repository.
+ 6. The integrated browser displays the editor to edit the problem statement.
+ 7. Orion also provides a button to |test-locally-button|, which will copy the local template or solution (whichever was selected) and the tests into a new folder and execute them locally.
+
+Orion for Tutors
+----------------
+
+Tutors can assess programming exercises via Orion by performing the following steps:
+
+ 1. Navigate to the exercise's assessment dashboard using the integrated browser
+ 2. There will be a button to |assess-in-orion-button|. It automatically downloads the automatic tests.
+ 3. After downloading or opening the exercise in IntelliJ, the normal submission overview is shown in the integrated browser. Each submission can be opened in Orion, new submissions can be started by |start-assessment-in-orion-button|. This downloads the submission files and overwrites the previous submission.
+ 4. The student's code is in the directory *assignment*. The tests are in the directory *tests*. Additionally, there are two directories *template* and *studentSubmission* that contain internal files and should be ignored.
```

Comment:
```suggestion
 4. The student's code is in the directory `assignment`. The tests are in the directory `tests`. Additionally, there are two directories `template` and `studentSubmission` that contain internal files and should be ignored.
```

Reply:
I personally prefer the italics, since it is not code

Reply:
Italics are supposed to be used for text that is less important than the main text. However, this is not the case here.
The inline code is my personal preference in markdown for such things but we could also make it bold.
Only italic I would not suggest due to the reasons mentioned.

Reply:
I disagree on the italics (e.g. [wikipedia](https://en.wikipedia.org/wiki/Italic_type) states: "Italics are a way to emphasise key points in a printed text, to identify many types of creative works, to cite foreign words or phrases, or, when quoting a speaker, a way to show which words they stressed.", but since other paths in the developer guide are also inline code, I'll do it as well.